test/clj_sqs_extended/serdes_test.clj

```diff
@@ -0,0 +1,33 @@
+(ns clj-sqs-extended.serdes-test
+  (:require [clojure.test :refer [deftest is testing]]
+            [clj-sqs-extended.serdes :as serdes]
+            [tick.alpha.api :as t]))
+
+
+(def ^:private basic-map
+  {:quote "State. You're doing it wrong."
+   :by "Rich Hickey"})
+
+(def ^:private timestamp-map
+  {:step 1
+   :timestamp (t/inst)})
+
+(deftest roundtrip-transit
+  (testing "Transit with basic map"
```

Comment:
I'd refactor up these different tests into their own top level tests. i.e. `roundtrip-transit-with-basic-map` and `roundtrip-transit-with-timestamp`